README.md

```diff
@@ -81,15 +85,22 @@ The following are examples of valid sentences:
 Write the BNF (not EBNF) description for this language.
 
 ## A2.1
-
-«replace this with your answer»
-
+<sentance>::="the"<adjective><noun><verb>|"the"<adjective><noun><verb><adverb>
+<adjective>::= ""|"lazy"|"smelly"|"lazy"<adjective>|"smelly"<adjective>
+<noun>::="dog"|"cat"
+<verb>::="ate"|"ran"
+<adverb>::="slowly"|"noisily"
```

Comment:
# 7

although line 89 would be better expressed as

~~~
<adjectives>::= "" | <adjectives> <adjective>
<adjective>::=  "lazy" | "smelly"
~~~

(ps. whitespace is your friend...)